### DISCUSSION TITLE
Connection over mobile hotspot

### DISCUSSION THREAD

**Discussion by Kareemic** [jomjol/AI-on-the-edge-device#2449] [Q&A] [answered]
so i have to bring the device to a place where normal WiFi network is not available (except SSID and password i have to set security to 802.1x Enterprise, phase2 to PAP, EAP method to TTLS and enter my username into identity field) is it possible to connect the device to such a network and if not is it possible to connect the device through a hotspot on my mobile?

**Comment by caco3** [comment, marked as answer]
The ESP Framework is quite picky about the Wifi to be used. so I would say you have to try it.

Alternatively, you could either
- Use it offline and just write the readings to the SD card, or
- use a 2nd device to monitor the USB log and use that one for your special wifi needs.


There are other users who use a 2nd device to monitor the USB Log and extract the data values from there.
